### PR TITLE
Contributing and support docs

### LINKED ISSUE #1791: Contributor guidelines missing
We should have contributor guidelines to help people get involved in development of the FlowKit project.

This is distinct from our existing developer documentation, and more along the lines of https://github.com/github/opensource.guide/blob/master/CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by greenape**
Closes #1791 

### I have:

- [ ] Formatted any Python files with [black](https://github.com/ambv/black)
- [x] Brought the branch up to date with master
- [x] Added any relevant Github labels
- [ ] Added tests for any new additions
- [x] Added or updated any relevant documentation
- [ ] Added an Architectural Decision Record (ADR), if appropriate
- [ ] Added an [MPLv2 License Header](https://www.mozilla.org/en-US/MPL/headers/) if appropriate
- [x] Updated the [Changelog](https://github.com/Flowminder/FlowKit/blob/master/CHANGELOG.md) 

### Description

Adds contributing and support docs to the `.github` folder.

### CODE REVIEW

**Comment by greenape** [comment]
I'm very open to tweaks (now or later) for either. Looking to achieve a good basis for what should be a living doc here.

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+Once a pull request is marked as ready, approved and all tests are passing, it will get merged automatically. The automatic merge process will also (try) to ensure the branch is up to date with master, but on occasion you may need to intervene to make sure.
+
+When preparing your branch for a pull request, it would be awesome if you took the time to clean up the history, and in general we prefer that you have kept up to date with master by rebasing as often as necessary.
+    
+## Reporting bugs :bug:
+
+We :heart: bug reports. Please have a quick look through the [issue tracker](https://github.com/Flowminder/FlowKit/issues?utf8=✓&q=is%3Aissue+is%3Aopen+label%3Abug+) to see if you're hitting something new, and [open up a new one](https://github.com/Flowminder/FlowKit/issues/new?template=bug_report.md) if you don't find anything.
+
+It is super helpful if you follow the template provided, but don't worry to much if you can't fill out all the fields.
```
Comment (jch1g10) [inline .github/CONTRIBUTING.md]:
```suggestion
It is super helpful if you follow the template provided, but don't worry too much if you can't fill out all the fields. 
```

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+## Requesting new features :rocket:
+
+Flowminder actively supports FlowKit, and we're always interested to hear suggestions for new features or enhancements. Check the [issue tracker](https://github.com/Flowminder/FlowKit/issues?q=is%3Aissue+is%3Aopen+label%3Aenhancement) to see if something is already on our radar and add a :+1: as a reaction if you think we should focus on it.
+
+You can also [request one](https://github.com/Flowminder/FlowKit/issues/new?template=feature_request.md) if nothing already listed matches what you have in mind, and get the conversation started.
+
+## Coding style
+
+For Python, we use [black](https://github.com/python/black), and for JS/JSX we use [prettier](https://github.com/prettier/prettier).
```
Comment (jch1g10) [inline .github/CONTRIBUTING.md]: We may also want to mention type-annotating arguments where possible, docstring format.

File: .github/SUPPORT.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+## Reporting bugs :bug:
+
+We :heart: bug reports. Please have a quick look through the [issue tracker](https://github.com/Flowminder/FlowKit/issues?utf8=✓&q=is%3Aissue+is%3Aopen+label%3Abug+) to see if you're hitting something new, and [open up a new one](https://github.com/Flowminder/FlowKit/issues/new?template=bug_report.md) if you don't find anything.
+
+It is super helpful if you follow the template provided, but don't worry to much if you can't fill out all the fields.
```
Comment (jch1g10) [inline .github/SUPPORT.md]:
```suggestion
It is super helpful if you follow the template provided, but don't worry too much if you can't fill out all the fields.
```

File: .github/SUPPORT.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+## Technical support
+
+We're happy to help out as much as we can - come find us on [Gitter](https://gitter.im/Flowminder/FlowKit), or drop as an e-mail at [flowkit@flowminder.org](mailto:flowkit@flowminder.org[?subject=GitHub]).
```
Comment (jch1g10) [inline .github/SUPPORT.md]:
```suggestion
We're happy to help out as much as we can - come find us on [Gitter](https://gitter.im/Flowminder/FlowKit), or drop us an e-mail at [flowkit@flowminder.org](mailto:flowkit@flowminder.org[?subject=GitHub]).
```

File: .github/SUPPORT.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+If you want more comprehensive support with using FlowKit, drop us an [email](mailto:flowkit@flowminder.org[?subject=Support]). and we can discuss how Flowminder can most effectively support your ambitions.
```
Comment (jch1g10) [inline .github/SUPPORT.md]:
```suggestion
If you want more comprehensive support with using FlowKit, drop us an [email](mailto:flowkit@flowminder.org[?subject=Support]) and we can discuss how Flowminder can most effectively support your ambitions.
```

**Pull request merged**